src/components/SuggestionsList/SuggestionsListItem.test.tsx

```diff
@@ -29,42 +31,21 @@ describe('<SuggestionsListItem />', () => {
 
   describe('when isSelected=true', () => {
     beforeEach(() => {
-      rendered = shallow(<SuggestionsListItem {...getProps({ isSelected: true })} />);
-    });
-
-    it('renders as expected', () => {
-      expect(rendered).toMatchSnapshot();
-    });
-  });
-
-  describe('when isSelected=false', () => {
-    beforeEach(() => {
-      rendered = shallow(<SuggestionsListItem {...getProps({ isSelected: false })} />);
-    });
-
-    it('renders as expected', () => {
-      expect(rendered).toMatchSnapshot();
-    });
-  });
-
-  describe('with avatarProps', () => {
-    beforeEach(() => {
-      const avatarProps = { borderType: BorderType.SOFT };
-      rendered = shallow(<SuggestionsListItem {...getProps({ avatarProps })} />);
+      component = shallow(<SuggestionsListItem {...baseProps({ isSelected: true })} item={textItem} />);
```

Comment:
Why not supply textItem to `baseProps` (weird naming IMO btw).

```ts
const props = getProps({ isSelected: true, item: textItem });
component = shallow(<SuggestionsListItem {...props} />);
```

Reply:
ya things changed when i moved the item from splat props to its own prop. i'll update

Reply:
done